packages/components/button/src/index.vue

```diff
@@ -10,6 +10,7 @@ const binds = Object.assign({}, useAttrs(), props.to ? { href: props.to } : {})
 
 <template>
   <component
+    :o="o"
```

Comment:
I dislike `o` property on component, i change it to `useAttrs`.
`const binds = Object.assign({}, useAttrs(), props.to ? { href: props.to } : {})`

How do you think? `o` property are too abstract, i think it works as a unocss variant, not a componet property. 
The same property has `border` ...

Reply:
But I think button should have its default style when I use <o-button>test</o-button>.

Reply:
> But I think button should have its default style when I use `<o-button>test</o-button>`.

const binds = Object.assign({o: 'primary' }, useAttrs(), props.to ? { href: props.to } : {}) 🤣